app/screens/home/Home.java

```diff
@@ -11,23 +12,31 @@ public class Home implements Screen {
     public void display() {
         Scanner sc = App.sc;
         int choice;
-        String[] options = { "Visit Stores", "Item Search", "Exit" };
+        ArrayList<String> options = new ArrayList<>();
+        options.add("Visit Stores");
+        options.add("Item Search");
+        options.add("Log-out");
 
         while (true) {
             MenuTitle.displayMainTitle();
             MenuTitle.displayOptions(options);
             System.out.print("Choice: ");
             try {
                 choice = Integer.parseInt(sc.nextLine());
-                if (choice == 1) {
-                    Router.navigate(SCREEN_ENUM.STORE_LIST);
-                } else if (choice == 2) {
-                    Router.navigate(SCREEN_ENUM.GLOBAL_SEARCH);
-                } else if (choice == 3) {
-                    return;
-                } else {
-                    System.out.println("Please choose from the given options...\n");
+                switch (choice) {
+                    case 1:
+                        Router.navigate(SCREEN_ENUM.STORE_LIST);
+                        break;
+                    case 2:
+                        System.out.println("Item Search");
+                        Router.navigate(SCREEN_ENUM.GLOBAL_SEARCH);
+                        break;
+                    case 3:
+                        return;
+                    default:
+                        System.out.println("Please choose from the given options...\n");
                 }
+
             } catch (Exception e) {
                 System.out.println("Please choose from the given options...\n");
```

Comment:
maybe change ang print here to print the message sa error?

`System.out.println(e.getMessage())`